lib/nuget-parser/csproj-parser.js

```diff
@@ -8,36 +8,35 @@ const _ = require('lodash');
 function getTargetFrameworksFromProjFile(rootDir) {
   debug('Looking for your .csproj file in ' + rootDir);
   const csprojPath = findFile(rootDir, /.*\.csproj$/);
-  if (!csprojPath) {
-    throw new Error('.csproj file not found in ' + rootDir + '. ' +
```

Comment:
is the only change here that you removed the throw? Hard to see in the diff

Reply:
correct. removed the throw at line 12 and 37